README.md

```diff
@@ -123,14 +123,25 @@ Generated files of the storybook go to the `storybook-static` folder.
 After cloning repository to your local, perform the following steps from the root of the repository.
 
 #### Steps:
-1. Build the Docker image:
+1. Development Setup with Docker:
+  1. Build the Docker image:
```

Comment:
This is not rendering in a proper way
Check the preview in online md file convertor